### PR TITLE
[Android] File in nnstreamer-ssd copy to nnstreamer-ssd-autocapture

### PR DESCRIPTION
**Pull request by JeongYeonUk**
nnstreamer-ssd-autocapture project is based on nnstreamer-ssd

Signed-off-by: JeongYeonUk <dusdnrl1@naver.com>

### CODE REVIEW

**Comment by helloahn** [comment]
[Template] should be removed. It's literally just a guide.

**Comment by JeongYeonUk** [comment]
> [Template] should be removed. It's literally just a guide.

Thank you!

File: android/example_app/nnstreamer-ssd-autocapture/jni/nnstreamer-ex.cpp [hunk 1]
```diff
@@ -0,0 +1,592 @@
+/**
+ * @file	nnstreamer-ex.cpp
+ * @date	1 April 2019
```
Comment (helloahn) [inline android/example_app/nnstreamer-ssd-autocapture/jni/nnstreamer-ex.cpp:3]: need to update the date & author

File: android/example_app/nnstreamer-ssd-autocapture/jni/nnstreamer-ex.cpp [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+ * @brief	Tensor stream example with TF-Lite model
```
Comment (helloahn) [inline android/example_app/nnstreamer-ssd-autocapture/jni/nnstreamer-ex.cpp:4]: The proper description is required. This `@brief` seems irrelevant to this example.

File: android/example_app/nnstreamer-ssd-autocapture/jni/nnstreamer-jni.c [hunk 3]
```diff
@@ -0,0 +1,732 @@
+/**
+ * @file	nnstreamer-jni.cpp
```
Comment (helloahn) [inline android/example_app/nnstreamer-ssd-autocapture/jni/nnstreamer-jni.c:2]: the doxygen update is required.

File: android/example_app/nnstreamer-ssd-autocapture/jni/nnstreamer-jni.h [hunk 4]
```diff
@@ -0,0 +1,54 @@
+/**
+ * @file	nnstreamer-jni.h
```
Comment (helloahn) [inline android/example_app/nnstreamer-ssd-autocapture/jni/nnstreamer-jni.h:2]: ditto

**Review by anyj0527 — COMMENTED** [review]
How about adding this new module in `settings.gradle` ?
